test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -393,6 +393,26 @@ public static String changeRoleBindingSubject(File roleBindingFile, String names
         }
     }
 
+    public static String changeDeploymentNamespaceUpgrade(File deploymentFile, String namespace) {
+        YAMLMapper mapper = new YAMLMapper();
+        try {
+            JsonNode node = mapper.readTree(deploymentFile);
+            // Change the docker org of the images in the 04-deployment.yaml
```

Comment:
You have if block where you match file name by  `"050-Deployment.*"` and here you have commented line with `04-deployment.yaml` is it just typo?